lib/fileHandler/index.js

```diff
@@ -1,23 +1,25 @@
-var fs     = require('fs');
-var util   = require('util');
-var PassThrough = require('stream').PassThrough;
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+const { PassThrough } = require('stream');
 
-var when   = require('when');
-var guid   = require('mout/random/guid');
-var mime   = require('mime');
-var moment = require('moment');
+const when = require('when');
+const guid = require('mout/random/guid');
+const mime = require('mime');
+const moment = require('moment');
 
-var needle = require('needle');
+const needle = require('needle');
 
-var AWS = require('aws-sdk');
+const AWS = require('aws-sdk');
 
-var REGION = 'us-west-2',
-	BUCKET = 'workflow-file-uploads';
+const REGION = 'us-west-2',
+	BUCKET = 'workflow-file-uploads',
+	DEV_BUCKET = 'workflow-file-uploads-dev';
```

Comment:
Should we consider creating our `workflow-file-uploads-dev` bucket in the same region as our dev environment (eu-west-2)? Even though I think the AWS console shows all regions, buckets do have a region setting I believe.

Reply:
yeah could be in eu-west-2

Reply:
Hmm, just tried setting the region to `eu-west-2`, but even though it worked, the signed URL didn't look like it normally does. Also, the URL still points to `us-west-2`.

Looking at the config Luka setup for this bucket, seems like it's configured for `us-west-2` - so will keep it as is (for now; can change later if needed).